zipkin-web/src/main/resources/app/js/page/dependency.js

```diff
@@ -24,22 +26,28 @@ define(
             dependenciesTemplate
   ) {
 
-    return initialize;
+    const DependencyPageComponent = component(function DependencyPage() {
+      this.after('initialize', function() {
+        window.document.title = 'Zipkin - Dependency';
+        this.trigger(document, 'navigate', {route: 'dependency'});
 
-    function initialize() {
-      window.document.title = 'Zipkin - Dependency';
-      $('.content').html(dependenciesTemplate());
+        this.$node.html(dependenciesTemplate());
 
-      const {startTs, endTs} = queryString.parse(location.search);
-      $('#endTs').val(endTs || moment().valueOf());
-      $('#startTs').val(startTs || moment().valueOf() - 7*24*60*60*1000); // set default startTs 7 days ago;
+        const {startTs, endTs} = queryString.parse(location.search);
+        $('#endTs').val(endTs || moment().valueOf());
+        $('#startTs').val(startTs || moment().valueOf() - 7*24*60*60*1000); // set default startTs 7 days ago;
```

Comment:
It seems to be configurable as a flag in zipkin-query. To be honest, I'm not 100 % sure what it does.